pythonforandroid/python.py

```diff
@@ -266,16 +267,19 @@ def create_python_bundle(self, dirn, arch):
                 '2' if self.version[0] == '2' else '',
                 self.major_minor_version_string
             ))
-        module_filens = (glob.glob(join(modules_build_dir, '*.so')) +
-                         glob.glob(join(modules_build_dir, '*.py')))
+        module_filens = list(glob.glob(join(modules_build_dir, '*.so')) +
```

Comment:
Is there any reason for calling `list()` when `glob.glob()` already returns a `list` and doing `[] + []` is implemented for lists?

Reply:
Oh yes, you are right, we don't need that...this commit was coming from an outdated branch of the python core...I don't know how I didn't see that  :sweat_smile: ... so many thanks @KeyWeeUsr  :blush: